src/parser/shaman/enhancement/modules/talents/LightningShield.js

```diff
@@ -48,29 +47,23 @@ class LightningShield extends Analyzer {
     return this.damageGained / (this.owner.fightDuration / 1000);
   }
 
-  get suggestionThresholds() {
-    return {
-      actual: this.selectedCombatant.getBuffUptime(SPELLS.LIGHTNING_SHIELD_TALENT.id),
-      isLessThan: {
-        major: 1,
-      },
-      style: 'decimal',
-    };
-  }
-
   suggestions(when) {
-    when(this.suggestionThresholds).addSuggestion((suggest, actual) => {
-      return suggest(<>You should fully utilize your <SpellLink id={SPELLS.LIGHTNING_SHIELD_TALENT.id} /> by using it before combat.</>)
-        .icon(SPELLS.LIGHTNING_SHIELD_TALENT.icon)
-        .actual(<>You kept up <SpellLink id={SPELLS.LIGHTNING_SHIELD_TALENT.id} /> for ${formatPercentage(actual)}% of the fight.</>)
-        .recommended(<>It is possible to keep up <SpellLink id={SPELLS.LIGHTNING_SHIELD_TALENT.id} /> for 100% of the fight by casting it pre-combat.</>);
-    });
+    const lightningUptime = this.selectedCombatant.getBuffUptime(SPELLS.LIGHTNING_SHIELD_TALENT.id) / this.owner.fightDuration;
+
+    when(lightningUptime).isLessThan(0.95)
```

Comment:
You should use suggestionThresholds for suggestions as it will then be consistent with other modules

Reply:
I didn't know that suggestionThresholds was used across the system. I returned the get function